Documentation build process is undocumented

The docs don't build, and the compile documentation doesn't provide much information on how they should be built.

Among other things this makes it harder to test the Maven snapshot release process.

@davecramer , I think you're probably the only one who knows the recipe for the secret sauce here.

```
$ ant doc
Buildfile: /home/craig/projects/2Q/pgjdbc/build.xml

prepare:

doc:
    [mkdir] Created dir: /home/craig/projects/2Q/pgjdbc/build/doc

BUILD FAILED
/home/craig/projects/2Q/pgjdbc/build.xml:755: stylesheet /home/craig/projects/2Q/pgjdbc/${docbook.stylesheet} doesn't exist.

Total time: 0 seconds
```

Presumably some additional properties must be set, pointing to stylesheets, possibly from packages that need to be installed?

There's a comment:

```
<!-- Blah. Can't reference an external XML catalog until Ant 1.6.
       So instead we copy the contents of ${docbook.dtd}/catalog.xml
       here, in the form that Ant's xmlcatalog element wants. -->
```

in `build.xml` that suggests perhaps this can be made easier now that we can sanely hard-depend on Ant 1.6.

Thoughts?

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,7 @@ before_script:
   - psql -c 'create database test owner test;' -U postgres
  
 
-script: ant -lib lib test
+script: ant test
 
 matrix:
   include:

--- a/build.xml
+++ b/build.xml
@@ -676,6 +676,12 @@ Import-Package: javax.sql, javax.transaction.xa, javax.naming, *;resolution:=opt
     <delete quiet="true" file="${srcdir}/${package}/ds/PGConnectionPoolDataSource.java" />
     <delete quiet="true" file="${srcdir}/${package}/xa/PGXADataSource.java" />
     <delete quiet="true" file="${srcdir}/${package}/ssl/MakeSSL.java" />
+    <delete quiet="true">
+      <fileset dir="lib">
+        <include name="*"/>
+        <exclude name="maven-ant-tasks*.jar"/>
+      </fileset>
+    </delete>
   </target>
 
   <!-- This compiles and executes the JUnit tests -->

--- a/build.xml
+++ b/build.xml
@@ -30,6 +30,9 @@
   <property file="build.local.properties" />
   <property file="build.properties"/>
 
+  <taskdef uri="antlib:org.apache.maven.artifact.ant"
+           classpath="lib/maven-ant-tasks-2.1.3.jar"/>
+
  <!-- define artifacts' name, which follows the convention of Maven -->
   <property name="maven.jar" value="${jardir}/${maven.artifact.id}-${maven.artifact.version}.jar"/>
   <property name="maven.javadoc.jar" value="${jardir}/${maven.artifact.id}-${maven.artifact.version}-javadoc.jar"/>

